java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+- Fix integer conversions for list of values
```

Comment:
Nitpick: this line needs more context for readers (or maybe merge with the one below?)